origami/tests/client_test.py

```diff
@@ -23,13 +23,13 @@ def extract_msg_transaction_id(msg):
     return UUID(json.loads(msg)['transaction_id'])
 
 
-@pytest.fixture
+@pytest_asyncio.fixture
```

Comment:
what's the reason to move to `pytest_asyncio` rather than `pytest`?

Reply:
Dependabot upgraded `pytest-asyncio` from 0.18.3 to 0.19.0 in this [PR](https://github.com/noteable-io/origami/pull/7) - and this release has some breaking changes around how async fixtures are implemented. More on that [here](https://github.com/pytest-dev/pytest-asyncio/releases) (Scroll to "Async fixtures")

This is the error I get by using `pytest.fixture` and not `pytest_asyncio.fixture`:
<img width="1459" alt="Screen Shot 2022-08-01 at 11 41 55 AM" src="https://user-images.githubusercontent.com/39169550/182200433-7240472b-e4b7-444c-b2fe-f24ba4b19e8d.png">